src/main/java/com/williamfiset/algorithms/sorting/QuickSort3.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Quicksort implementation using Hoare partitioning
```

Comment:
I think your description in https://github.com/williamfiset/Algorithms/issues/183 is more accurate?

Reply:
Ohh yeah sorry I'll do it right away.